CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # CHANGELOG
 
 
+## vN.N.N
```

Comment:
@nanetteCodes - please give it a version number and describe in detail of what fields (with type, name, etc.) are added or deleted like this: https://github.com/triniti/schemas/commit/1c019bb647531123b81b6291b5476ef696e04c6f#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR4